core/src/dird/jcr_util.h

```diff
@@ -0,0 +1,32 @@
+/*
+   BAREOS® - Backup Archiving REcovery Open Sourced
+
+   Copyright (C) 2018-2020 Bareos GmbH & Co. KG
+
+   This program is Free Software; you can redistribute it and/or
+   modify it under the terms of version three of the GNU Affero General Public
+   License as published by the Free Software Foundation and included
+   in the file LICENSE.
+
+   This program is distributed in the hope that it will be useful, but
+   WITHOUT ANY WARRANTY; without even the implied warranty of
+   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+   Affero General Public License for more details.
+
+   You should have received a copy of the GNU Affero General Public License
+   along with this program; if not, write to the Free Software
+   Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA
+   02110-1301, USA.
+*/
+
+#ifndef BAREOS_DIRD_JCR_UTIL_H_
+#define BAREOS_DIRD_JCR_UTIL_H_
+
+#include "dird/jcr_private.h"
+
+namespace directordaemon {
+
+JobControlRecord* InitJcrConfig(JCR_free_HANDLER* DirdFreeJcr);
```

Comment:
```suggestion
JobControlRecord* NewDirectorJcr(JCR_free_HANDLER* DirdFreeJcr);
```
As the functionality was not changed, please keep the name of the function.